### PR TITLE
Feature friends routes

### PR DESCRIPTION
**Pull request by alexarlington**
## Changes
Added all the friends routes and tests

## Issue (if any)

The issue this PR solves is: #38 
(type # and the number of the issue to automatically create a link)

## Checklist
I have:

- [x] Run this code locally to make sure that it works
- [x] Written tests for _all_ relevant functionality
- [x] Ensured it works with latest dev (run `git rebase origin/dev` to get latest dev into your branch)
- [x] Included a link to the issue / task / project card this PR solves*
- [x] Ensured you are creating a pull request to DEV not MAIN
- [x] Notified one of the lead devs of this PR

### CODE REVIEW

File: server/routes/friends.test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    .expect(200)
+    .then((response) => {
+      // console.log('FRIENDS: ', response.body)
+      expect(response.body.friends).toHaveLength(3)
+      return null
+    })
+})
+
+test('POST /user_id adds a new friend', () => {
+  db.addFriend = jest.fn()
```
Comment (alexarlington) [inline server/routes/friends.test.js:26]: we need to remove this line when the friends db funcitons are ready

File: server/routes/friends.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    .send(newFriend)
+    .expect(200)
+    .then((response) => {
+      expect(response.status).toBe(200)
+      return null
+    })
+})
+
+test('DELETE deletes user from following list', () => {
+  db.deleteFriend = jest.fn()
```
Comment (alexarlington) [inline server/routes/friends.test.js:42]: we need to remove this line when the friends db funcitons are ready

File: client/components/App.test.js [hunk 3]
```diff
@@ -1,32 +1,32 @@
-import React from 'react'
```
Comment (alexarlington) [inline client/components/App.test.js:1]: Commented out this whole file as per instruciton from Jared

File: server/routes/media.js [hunk 4]
```diff
@@ -4,6 +4,8 @@ const db = require('../db/media')
 
 const router = express.Router()
 
+// GET /api/v1/media/getMedia/id
```
Comment (BrenMurrell) [inline server/routes/media.js:7]: Bren's gonna get rid of these :)

File: server/routes/media.test.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+const request = require('supertest')
```
Comment (BrenMurrell) [inline server/routes/media.test.js:1]: This file shouldn't be in this PR - irrelevant to feature

**Review by BrenMurrell — CHANGES_REQUESTED** [review]
hey can you please check these media files - shoudln't be here

(Bren will fix)

File: server/routes/media.js (hunk 4 above)
Comment (BrenMurrell) [reply server/routes/media.js]: done!

**Review by BrenMurrell — APPROVED** [review]
Approved! 🚀

**Pull request merged**